core/src/main/java/org/apache/brooklyn/entity/group/DynamicClusterImpl.java

```diff
@@ -825,8 +858,12 @@ protected Entity replaceMember(Entity member, @Nullable Location memberLoc, Map<
             addedEntities.add(entity);
             addedEntityLocations.put(entity, loc);
             if (entity instanceof Startable) {
+                boolean privileged = false;
+                if (Boolean.TRUE.equals(entity.sensors().get(AbstractGroup.FIRST_MEMBER))) {
```

Comment:
Remind me why not just `privileged = entity.sensors().get(AbstractGroup.FIRST_MEMBER)`?